analyses/molecular-subtyping-embryonal/results/cleaned_chr19_cn.tsv

```diff
@@ -7,8 +7,7 @@ BS_AQMKA8NC	No
 BS_B73JK7T1	No
 BS_BFDEZK1C	No
 BS_JC8YV5V9	No
-BS_JX1PK0VW	No
-BS_K07KNTFY	Yes
+BS_K07KNTFY	No
```

Comment:
BS_K07KNTFY has lost it's chr 19 amplification
```
cnv_consensus %>% filter(chrom == "chr19",
+          seg.mean > 0,ID %in% emb_id$Kids_First_Biospecimen_ID)
# A tibble: 2 x 7
  ID          chrom loc.start  loc.end num.mark seg.mean copy.num
  <chr>       <chr>     <dbl>    <dbl> <lgl>       <dbl>    <dbl>
1 BS_69VS8PS1 chr19  53653251 54427564 NA           1.62        5
2 BS_TE8QFF7T chr19  53659169 54424398 NA           1.76        8
```

but checking controlfreec and cnvkit it seems there is an amplification
```
freec %>% filter(grepl("19",cytoband),status !="loss",
+          biospecimen_id %in% "BS_K07KNTFY")
# A tibble: 1,591 x 7
   biospecimen_id status copy_number ploidy ensembl       gene_symbol cytoband
   <chr>          <chr>        <dbl>  <dbl> <chr>         <chr>       <chr>   
 1 BS_K07KNTFY    gain             5      4 ENSG00000283… MIR1302-11  19p13.3 
 2 BS_K07KNTFY    gain             5      4 ENSG00000282… WASH5P      19p13.3 
 3 BS_K07KNTFY    gain             5      4 ENSG00000282… FAM138F     19p13.3 
 4 BS_K07KNTFY    gain             5      4 ENSG00000176… OR4F17      19p13.3 
 5 BS_K07KNTFY    gain             5      4 ENSG00000141… PLPP2       19p13.3 
 6 BS_K07KNTFY    gain             5      4 ENSG00000099… MADCAM1     19p13.3 
 7 BS_K07KNTFY    gain             5      4 ENSG00000141… TPGS1       19p13.3 
 8 BS_K07KNTFY    gain             5      4 ENSG00000099… CDC34       19p13.3 
 9 BS_K07KNTFY    gain             5      4 ENSG00000197… GZMM        19p13.3 
10 BS_K07KNTFY    gain             5      4 ENSG00000172… BSG         19p13.3 
# … with 1,581 more rows
> cnvkit %>% filter(grepl("19",cytoband),status !="loss",
+          biospecimen_id %in% "BS_K07KNTFY")
# A tibble: 80 x 7
   biospecimen_id status copy_number ploidy ensembl       gene_symbol cytoband
   <chr>          <chr>        <dbl>  <dbl> <chr>         <chr>       <chr>   
 1 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR512-1    19q13.42
 2 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR512-2    19q13.42
 3 BS_K07KNTFY    gain             6      4 ENSG00000221… MIR1323     19q13.42
 4 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR498      19q13.42
 5 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR520E     19q13.42
 6 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR515-1    19q13.42
 7 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR519E     19q13.42
 8 BS_K07KNTFY    gain             6      4 ENSG00000283… MIR520F     19q13.42
 9 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR515-2    19q13.42
10 BS_K07KNTFY    gain             6      4 ENSG00000207… MIR519C     19q13.42
# … with 70 more rows
```